README.md

```diff
@@ -1148,6 +1149,20 @@ $ make run-smoke-tests [key=value]*
 It will run only tests tagged with `@smoke`.
 All options from BDD tests do also apply here.
 
+#### Running performance tests
+
+The BDD tests also provide performance tests. These tests are ignored unless you
+specifically provide the `@performance` tag or run:
+
+```bash
+$ make run-performance-tests [key=value]*
+```
+
+It will run only tests tagged with `@performance`. 
+All options from BDD tests do also apply here.
+
+**NOTE:** Performance tests should be run without concurrency.
```

Comment:
could we just disable concurrency into the code directly if performance is enabled ?
would just need to update `opt.Concurrency` to 1 into main_test.

Reply:
Well, if we keep concurrent 1 in Makefile like it is now I think it's OK. Wdyt?

Reply:
I think we should force it in Go code or ?
so we avoid any problem, whatever the user put as concurrency, it stays at 1.

Reply:
To be honest I am also fine with the current way. I anyway want to have it as a Jenkins input parameter so I know that for 100% I am passing there 1.

Reply:
oki